components/molecule/select/src/index.scss

```diff
@@ -40,6 +43,13 @@ $bgc-atom-input-disabled: $c-gray-lightest !default;
         cursor: default;
         #{$class-select-atom-input-input} {
           background: $bgc-atom-input-disabled;
```

Comment:
same thing as `$bd-molecule-select-disabled` is for `$bgc-atom-input-disabled`. plzz